docs/sources/project/test-and-docs.md

```diff
@@ -230,6 +230,42 @@ with new memory settings.
 6. Restart your container and try your test again.
 
 
+## Testing just the Windows client
+
+This explains how to test the Windows client on a Windows server set up as a development environment.
+
+
+1.  If you don't have one, start a Git Bash window.
```

Comment:
Sorry should have been clearer - we're using a proper noun to describe something - I wanted to confirm we explain how to do it or that is was a logical process for someone.